public/scss/_forms.scss

```diff
@@ -78,11 +62,6 @@ form.pure-form.pure-form-stacked {
     }
   }
 
-  button {
-    @extend %button-styles;
-    font-family: 'Noto Sans', sans-serif;
-  }
-
   .buttons {
     display: flex;
```

Comment:
👌 but what kinda browser support we looking at? Might need some fallback styles if we're doing .
IE 10 needs the `ms-` prefix, IE 11 we'll just have to double check because it can get weird, and IE 9 support is 😬

Reply:
Currently, we haven't discussed browser support. 

My opinion is that we want IE11 for sure, and maybe IE10.

We have access to [browserstack](https://www.browserstack.com/), so we can test with different browsers, but I think we should leave this until next sprint.